src/catalog/src/durable/transaction.rs

```diff
@@ -101,6 +103,7 @@ pub struct Transaction<'a> {
     commit_ts: mz_repr::Timestamp,
     /// The ID of the current operation of this transaction.
     op_id: Timestamp,
+    network_policies: TableTransaction<NetworkPolicyKey, NetworkPolicyValue>,
```

Comment:
nit: I would order this above the `commit_ts` and `op_id`, but it doesn't really matter